### ISSUE TITLE
how to use the model created 

### ISSUE THREAD

**Issue by fuyiqin** [dgurkaynak/marvel-finetuning#15]
I want to use the model created to  extract the features of the image，can you tell me how to use your code

**Comment by dgurkaynak** [comment]
See the prediction code: https://github.com/dgurkaynak/tensorflow-cnn-finetune/issues/4#issuecomment-384371453

If you want to just extract the feature, not the prediction stuff, you can get directly the output of model instead of accuracy, like:
```
features = sess.run(model.prob, feed_dict={x: batch_tx, y: batch_ty, is_training: False})      
```
